src/parser/dotnet_six.rs

```diff
@@ -0,0 +1,85 @@
+use crate::models::{Log, LogLevel, RawCloudWatchLog, StructuredLog};
+use recap::Recap;
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize, Recap)]
+#[recap(regex = r#"(?x)
+    (?P<timestamp>\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z))
+    \s+
+    (?P<guid>[0-9A-Fa-f]{8}[-][0-9A-Fa-f]{4}[-][0-9A-Fa-f]{4}[-][0-9A-Fa-f]{4}[-][0-9A-Fa-f]{12})
+    \s+
+    (?P<level>(info)|(warn)|(error))
+    \s+
+    (?P<data>(?s).*)
+  "#)]
+struct DotnetSixCloudWatchLog {
+    timestamp: String,
+    guid: String,
+    level: String,
+    data: String,
+}
+
+impl Into<StructuredLog> for DotnetSixCloudWatchLog {
+    fn into(self) -> StructuredLog {
+        StructuredLog {
+            timestamp: Some(self.timestamp),
+            guid: Some(self.guid),
+            level: match self.level.as_str() {
+                "info" => Some(LogLevel::Info),
+                "warn" => Some(LogLevel::Warn),
+                "error" => Some(LogLevel::Error),
```

Comment:
It looks like there are other levels: Trace, Critical etc. https://github.com/aws/aws-lambda-dotnet/blob/master/Libraries/src/Amazon.Lambda.Core/ILambdaLogger.cs#L7

Should we support these here as well?